README.md

```diff
@@ -111,6 +111,41 @@ grep extra_css README.md && exit 2
 ```
 ````
 
+### Caching
+
+Speed up your builds by caching execution results:
+
+````md
+```python exec="yes" cache="yes"
+# Expensive computation
+import time
+time.sleep(5)
+print("Done!")
+```
+````
+
+Use custom cache IDs for persistence across builds:
+
+````md
+```python exec="yes" cache="my-plot"
+# Generate plot - will be cached
+import matplotlib.pyplot as plt
+# ...
+```
+````
+
+Force cache refresh with `refresh="yes"`:
```

Comment:
I'm not convinced by the utility of this refresh feature. Having to update the code block, build docs, then modify the code block back seems tedious. I'd prefer a solution where an environment variable is used, as for global refreshes. This would require IDs of course, so, to be discussed.

Generally speaking, this feature doesn't seem to come from the requirements identified in #4. It doesn't address what was said there.